kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java

```diff
@@ -168,6 +175,7 @@ protected void onLookupComplete(ChannelHandlerContext ctx, Future<SslContext> fu
     void addHandlers(SocketChannel ch, VirtualClusterBinding binding) {
         var virtualCluster = binding.virtualClusterModel();
         ChannelPipeline pipeline = ch.pipeline();
+        pipeline.remove(LOGGING_INBOUND_ERROR_HANDLER_NAME);
```

Comment:
is this to move it from the tail of the pre-resolution pipeline to the tail of the post resolution pipeline?

Reply:
yep, I want it to always be last to catch any unhandled exceptions that make it to the end of the inbound pipe.

Initially I only added it once and then used a more awkward `addBefore` call everywhere else to put the new handlers behind the error handler, but it made it harder to understand.